tidb-cloud/tidb-cloud-auditing.md

```diff
@@ -65,7 +65,7 @@ For more information, see [Creating a bucket](https://docs.aws.amazon.com/Amazon
     - If yes, record the matched storage bucket policy for later use.
     - If not, go to **IAM** > **Access Management** > **Policies** > **Create Policy**, and define a bucket policy according to the following policy template.
 
-        ```json
+        ```
```

Comment:
Just temporarily remove "json" to fix the color issue of this page. I'll add it back before merging this PR.